daemon/config/config.go

```diff
@@ -514,6 +515,12 @@ func getConflictFreeConfiguration(configFile string, flags *pflag.FlagSet) (*Con
 		return nil, err
 	}
 
+	if runtime.GOOS != "windows" {
+		if _, ok := config.Features["windows-no-dns-proxy"]; ok {
```

Comment:
Bike-shedding; would `(windows)-disable-dns-proxy` be clearer?

Also (question); is this an option we could see becoming opt-in/opt-out for non-Windows? And in that case, do we need the `windows-` prefix? (we can still document - and error- when used on other platforms)

Reply:
On the name ... they look quite equivalent to me (and the current name is less typing!), but happy to change it if you like?

It started off as a Windows and Linux option - but there's no known use-case for disabling it on Linux, and we don't really need more options to maintain. It's really only meant as an emergency off-switch ... hopefully nobody will need it but, if there's something we didn't think of, being able to restore the old behaviour might save us from having to make a patch in a hurry. So, Albin suggested making it a Windows-only feature flag - which will make it easier to remove in future if it's not useful.